code/modules/martial_arts/martial.dm

```diff
@@ -8,6 +8,7 @@
 	var/temporary = FALSE
 	var/datum/martial_art/base = null // The permanent style
 	var/deflection_chance = 0 //Chance to deflect projectiles
+	var/reroute_deflection = FALSE // Can it reflect projectiles in a random direction?
```

Comment:
Please use the proper [comment format for vars](https://github.com/ParadiseSS13/Paradise/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine), `///`